connectors/migrations/20241205_check_confluence_modified_pages.ts

```diff
@@ -0,0 +1,60 @@
+import { makeScript } from "scripts/helpers";
+
+import {
+  fetchConfluenceConfigurationActivity,
+  getConfluenceClient,
+  getSpaceIdsToSyncActivity,
+} from "@connectors/connectors/confluence/temporal/activities";
+import { ConnectorResource } from "@connectors/resources/connector_resource";
+
+makeScript(
+  {
+    timeWindowMs: {
+      type: "number",
+      demandOption: false,
+      default: 60 * 60 * 1000,
+      description: "Number of connectors to process concurrently",
+    },
+  },
+  async ({ timeWindowMs }) => {
+    const connectors = await ConnectorResource.listByType("confluence", {});
+
+    for (const connector of connectors) {
+      console.log(`\n -- Checking connector ${connector.id}`);
+      let connectorCount = 0;
+
+      const confluenceConfig = await fetchConfluenceConfigurationActivity(
+        connector.id
+      );
+      const { cloudId } = confluenceConfig;
+
+      const client = await getConfluenceClient({
+        cloudId,
+        connectorId: connector.id,
+      });
+
+      const spaceIds = await getSpaceIdsToSyncActivity(connector.id);
+
+      for (const spaceId of spaceIds) {
+        const { pages } = await client.getPagesInSpace(
```

Comment:
So we only take the first 25 pages. If we want a full picture we might want to do a `do/while`, no?

Reply:
done